sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xmlns:eco="http://rs.tdwg.org/eco/terms/"
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title='Humboldt Ecological Inventory'
+    name="HumboldtEcologicalInventory" namespace='http://rs.tdwg.org/eco/terms/'
+    rowType='http://rs.tdwg.org/dwc/terms/Event'
```

Comment:
As per last time @tucotuco - this can't be Event.

The IPT keys the data schemas on the `rowType` and the file represents the schema for the type. Event is already defined elsewhere, so this either needs to be in a new namespace or called something else.